modules/updating-lmeval-job-configuration-using-the-console.adoc

```diff
@@ -0,0 +1,81 @@
+:_module-type: PROCEDURE
+
+ifdef::context[:parent-context: {context}]
+[id="updating-lmeval-job-configuration-using-the-console_{context}"]
+= Updating LMEval Job configuration using the console
+
+[role='_abstract']
+Follow these steps to enable online access (`allowOnline`) and remote code execution (`allowCodeExecution`) modes through the {productname-short} console for LMEval Jobs. 
+
+[IMPORTANT]
+====
+Enabling online access or code execution involves a security risk. Only use these configurations if you trust the source(s).
+====
+
+.Prerequisites
+
+* You have cluster administrator privileges for your {productname-long} cluster.
+
+.Procedure
+. Navigate to *Workloads → ConfigMaps*.
+
+. Select the *`redhat-ods-applications`* namespace.
+
+. Open the *`trustyai-service-operator-config`* ConfigMap.
+
+. Under *Annotations*, add:
++
+[source, yaml]
+----
+opendatahub.io/managed: "false"
+----
+. Under *Data*, add or update:
++
+[source, yaml]
+----
+lmes-allow-online: "true"
+lmes-allow-code-execution: "true"
+----
+. Save the ConfigMap.
+
+. Navigate to *Workloads → Deployments*.
+
+. Select the *`redhat-ods-applications`* namespace.
+
+. Locate *`trustyai-service-operator-controller-manager`*.
+
+. Click the *Actions* menu and select *Restart rollout*.
++
+[NOTE]
+--
+Each LMEval Job must explicitly be set to allow online access and code execution.
+--
+. Add the following fields to the `LMEvalJob` to enable online access and code execution for this job:
```

Comment:
I think this wording might imply you already have created an LMEvalJob- maybe something like:

When writing your `LMEvalJob` custom resource, ensure that the following fields are set to 'true' to enable  online access and code execution for this job: